api/BeforeUnloadEvent.json

```diff
@@ -21,7 +21,7 @@
             "version_added": "4"
           },
           "ie": {
-            "version_added": true
+            "version_added": "9"
```

Comment:
The IE 4 from there came from the wiki tables, so I'm not entirely certain on its accuracy.  Additionally, I had determined that the Event API was implemented in IE 6.  I'll see if I can get to the bottom of this.

Reply:
This is a pretty old API I think, so I wouldn't be surprised if it's older than IE9. A test using both of the forms in https://developer.mozilla.org/en-US/docs/Web/API/WindowEventHandlers/onbeforeunload#syntax should reveal when this event is fired, and probably it was a special kind of event since the beginning, but logging the event should reveal that.